gui/resources.xml

```diff
@@ -459,6 +469,7 @@ Vocabulary
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Add a new layer group below the current layer.</property>
           <signal name="activate" handler="new_layer_cb"/>
         </object>
+        <accelerator key="g" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
```

Comment:
Yeah, your right. When I create a group layer I always tend to make it above the active layer than below it. I'll revert it.